app/middlewares/user.js

```diff
@@ -0,0 +1,16 @@
+const { check } = require('express-validator/check');
+
+export function createValidation() {
```

Comment:
You should use exports here, ecmascript module syntax is not supported in node LTS 12.x